### PR TITLE
refactor(tests): Remove superfluous imports

### PR DESCRIPTION
**Pull request by GSVarsha**
Follow up PR for #503

### CODE REVIEW

File: instana/collector/base.py [hunk 1]
```diff
@@ -13,10 +13,7 @@
 from ..util import every, DictionaryOfStan
 
 
-if sys.version_info.major == 2:
-    import Queue as queue
-else:
-    import queue # pylint: disable=import-error
+import queue # pylint: disable=import-error
```
Comment (Ferenc-) [inline instana/collector/base.py:16]: Is the comment needed?

File: instana/util/ids.py [hunk 2]
```diff
@@ -11,13 +10,7 @@
...
 BAD_ID = "BADCAFFE"  # Bad Caffe
 
-PY2 = sys.version_info[0] == 2
-PY3 = sys.version_info[0] == 3
-
-if PY2:
-    string_types = basestring
-else:
-    string_types = str
+string_types = str
```
Comment (Ferenc-) [inline instana/util/ids.py:13]: I think we should just use str everywhere?

**Review by Ferenc- — CHANGES_REQUESTED** [review]
Looking good. Perhaps a few extra lines could be shedded.

File: instana/collector/base.py (hunk 1 above)
Comment (GSVarsha) [reply instana/collector/base.py:16]: Keeping the `pylint` comment as discussed.

**Review by Ferenc- — APPROVED** [review]
LGTM

**Review by pvital — APPROVED** [review]
It looks good to me.

**Pull request merged**